File: app/src/main/java/com/singularity/trackmyvehicle/model/apiResponse/v2/LoginResponse.java
package com.singularity.trackmyvehicle.model.apiResponse.v2;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Sadman Sarar on 3/11/18.
 */

public class LoginResponse {

  @Expose
  @SerializedName("user")
  public User user;
  @Expose
  @SerializedName("access_time")
  public String accessTime;
  @Expose
  @SerializedName("access_token")
  public String accessToken;
  @Expose
  @SerializedName("context")
  public String context;
  @Expose
  @SerializedName("user_message")
  public String userMessage;
  @Expose
  @SerializedName("app_message")
  public String appMessage;
  @Expose
  @SerializedName("code")
  public int code;

  public static class User {
  	public String id;
    @Expose
    @SerializedName("phone")
    public String phone;
    @Expose
    @SerializedName("email")
    public String email;
    @Expose
    @SerializedName("name")
    public String name;
    public String userGroupIdentifier = "";
  }
}

File: app/src/main/java/com/singularity/trackmyvehicle/model/apiResponse/v3/LoginResponse.kt
package com.singularity.trackmyvehicle.model.apiResponse.v3

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Imran Chowdhury on 2020-01-21.
 */

data class UserResponseWrapper(
        @Expose
        @SerializedName("User")
        var login: LoginResponse? = null)
data class TerminalListResponseWrapper(
        @Expose
        @SerializedName("Terminal")
        var data: List<Terminal>? = null)

data class LoginResponse(
        @Expose
        @SerializedName("UserID")
        var userId: String? = null,
        @Expose
        @SerializedName("UserEmail")
        var userEmail: String? = null,
        @Expose
        @SerializedName("UserSignInName")
        var userSignInName: String? = null,
        @Expose
        @SerializedName("UserPasswordHashSalt")
        var userPasswordHashSalt: String? = null,
        @Expose
        @SerializedName("UserName")
        var userName: String? = null,
        @Expose
        @SerializedName("UserPhone")
        var userPhone: String? = null,
        @Expose
        @SerializedName("UserGroupIdentifier")
        var userGroupIdentifier: String? = null,
        @Expose
        @SerializedName("UserGroupName")
        var userGroupName: String? = null,
        @Expose
        @SerializedName("UserGroupWeight")
        var userGroupWeight: String? = null
)